src/config/default.yml

```diff
@@ -62,50 +62,45 @@ try:
   # XXX: This url is temporary until a real one hits mozilla-central.
   errorTaskUrl: "https://gist.githubusercontent.com/gregarndt/8fc123be2408180c2f13/raw/d12f500faa334bc72cb1dd033e028773ac2476c8/error.yml"
 
-  # Default scopes to use for projects which have no specific scopes of their own.
-  defaultScopes:
```

Comment:
Either we need to update this [1] or make default scopes and empty array?

[1] https://github.com/taskcluster/mozilla-taskcluster/blob/master/src/config.js#L94